games-arcade/caveexpress/caveexpress-2.4.recipe

```diff
@@ -0,0 +1,68 @@
+SUMMARY="Classic 2D platformer with physics-based gameplay and dozens of levels"
+DESCRIPTION="Master your pedal-powered flying machine to pick up packages \
+from your cave-dwelling clients and drop them off at the collection point. \
+But beware! Mighty mastodons, terrifying pterodactyls and others \
+would rather see you extinct."
+HOMEPAGE="https://caveproductions.org/"
+COPYRIGHT="2014-2016 Martin Gerhardy"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/mgerhardy/caveexpress/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="c1bf0707a4a2bcf2358b2791d97674a790d9fdce3ba29c695b9fa97030d74d3f"
+
+PATCHES="caveexpress-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	caveexpress$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libatomic$secondaryArchSuffix
+	lib:libBox2D$secondaryArchSuffix
+	lib:liblua$secondaryArchSuffix
+	lib:libncurses$secondaryArchSuffix
+	lib:libSDL2_2.0$secondaryArchSuffix
+	lib:libSDL2_image_2.0$secondaryArchSuffix
+	lib:libSDL2_mixer_2.0$secondaryArchSuffix
+	lib:libSDL2_net_2.0$secondaryArchSuffix
+	lib:libsqlite3$secondaryArchSuffix
+	lib:libyajl$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libBox2D$secondaryArchSuffix
+	devel:liblua$secondaryArchSuffix
+	devel:libSDL2_2.0$secondaryArchSuffix
+	devel:libSDL2_image_2.0$secondaryArchSuffix
+	devel:libSDL2_mixer_2.0$secondaryArchSuffix
+	devel:libSDL2_net_2.0$secondaryArchSuffix
+	devel:libncurses$secondaryArchSuffix
+	devel:libsqlite3$secondaryArchSuffix
+	devel:libyajl$secondaryArchSuffix
+	glm${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:lua$secondaryArchSuffix
+	cmd:luac$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	mkdir -p build && cd build
+	cmake .. -DCMAKE_INSTALL_PREFIX=$appsDir
+	make install
+}
+
+INSTALL()
+{
```

Comment:
make install

Reply:
When I'm trying to build it says "no rule to make target 'install'"

Reply:
You need to change directory:
```diff
+	cd build
+	make install
```

Reply:
Oh, thanks I miss it.